lang/cs/Org.Apache.REEF.Tests/Functional/Common/EventHandle.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System.Threading;
+using Org.Apache.REEF.Tang.Annotations;
+
+namespace Org.Apache.REEF.Tests.Functional.Common
+{
+    /// <summary>
+    /// An test EventHandle that simply wraps around a <see cref="ManualResetEventSlim"/>.
+    /// </summary>
+    public sealed class EventHandle
```

Comment:
maybe not, I assumed it should be EventHandler (r)

---
In reply to: [67907173](https://github.com/apache/reef/pull/1042#discussion_r67907173) [](ancestors = 67907173)